pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md

```diff
@@ -0,0 +1,376 @@
+      }
+    },
+    {
+      "ID": "rule2",
+      "Status": "Enabled",
+      "Destination": {
+        "Bucket": "arn:aws:s3:::region2-destination-bucket"
+      }
+    }
+  ]
+}
+```
+Suppose the source bucket, region1-destination-bucket and region2-destination-bucket are 3 buckets in 3 OVHcloud regions, this configuration will allow you to backup all objects in the source bucket to 2 different regions.
+
+#### Replicating 2 subsets of objects to different destination buckets;
+
+```json
+{
+  "Role": "IAM-role-ARN",
+  "Rules": [
+    {
+      "ID": "rule1",
+      "Status": "Enabled",
+      "Filter" : {
+        "Prefix": "dev"
+      },
+      "Destination": {
+        "Bucket": "arn:aws:s3:::destination-bucket1"
+      },
+      "DeleteMarkerReplication": { "Status": "Disabled" },
+    },
+    {
+      "ID": "rule2",
+      "Status": "Enabled",
+      "Filter" : {
+        "Prefix": "prod"
+      },
+      "Destination": {
+        "Bucket": "arn:aws:s3:::destination-bucket2"
+      },
+      "DeleteMarkerReplication": { "Status": "Disabled" }
+    }
+  ]
+}
+```
+
+### This configuration contains 2 replication rules:
+
+- rule1 will replicate all objects with prefix "dev" to bucket destination-bucket1 and additionally, will replicate also deletion operations.
+- rule2 will replicate all objects with prefix "prod" to bucket destination-bucket2 without replicating deletion operations.
+
+## In Practice
+
+
+> [!warning]
+> Versioning must be activated in source bucket and destination bucket(s)
+
+### Via the cli
+#### Create source bucket
+The source bucket is the bucket whose objects are automatically replicated.
+
+```bash
+$ aws s3 mb s3://<bucket_name>
+aws --endpoint-url https://s3.<region_in_lowercase>.<storage_class>.cloud.ovh.net --profile default s3 mb s3://<bucket_name>
+```
+**_Example:_** Creation of a source bucket in the SBG region in the Standard storage class.
+
+```bash
+$ aws s3 mb s3://my-source-bucket
+aws --endpoint-url https://s3.sbg.io.cloud.ovh.net --profile default s3 mb s3://my-source-bucket
+```
+
+#### Activate versioning in destination bucket
+```bash
+$ aws --endpoint-url https://s3.<region_in_lowercase>.<storage_class>.cloud.ovh.net --profile default s3api put-bucket-versioning --bucket my-destination-bucket --versioning-configuration Status=Enabled
+```
+#### Apply replication configuration
+Using the AWS cli, replication configuration is applied on the source bucket.
+```bash
+$ aws --endpoint-url https://s3.gra.io.cloud.ovh.net --profile default s3api put-bucket-replication --bucket <source> --replication-configuration <conf.json>
+```
+
+**_Example:_**: Replicate all objects with prefix "docs" having a tag "importance" with value "high" to "my-destination-bucket" and replicate the delete markers i.e objects marked as deleted in source will be marked as deleted in destination
+
+```bash
+{
+   "Role": "arn:aws:iam::<your_project_id>:role/replicationRole",
```

Comment:
`replicationRole` must be `s3-replication`.
Otherwise, the replication won't work (it corresponds to a special role).